index.html

```diff
@@ -774,24 +774,34 @@ <h3>DID Syntax</h3>
 
       <div class="note" title="Persistence">
         <p>
-  A <a>DID</a> is expected to be persistent and immutable. That is, a <a>DID</a>
-  is bound exclusively and permanently to its one and only subject. Even after a
-  <a>DID</a> is deactivated, it is intended that it never be repurposed.
+The persistence of a <a>DID</a>, i.e., the ability for it to continue to
+identify the same <a>DID subject</a> over time, is a function of: a) the DID
+controller, and b) the <a>DID method</a>. While <a>DID</a> architecture is
+designed to support the ability for a <a>DID</a> to be permanently bound to one
+<a>DID subject</a> for all time, there are two important caveats:
         </p>
+        <ol start="1">
+          <li>
+The <a>DID controller</a> may wish to terminate this binding—or possibly even
+bind the <a>DID</a> to a different <a>DID subject</a> at some point in time.
+          </li>
+          <li>
+Even if a permanent binding is desired, maintaining this binding is dependent
+on the infrastructure required by the <a>DID method</a>.
+          </li>
+        </ol>
         <p>
-Ideally, a <a>DID</a> would be a completely abstract decentralized identifier
-(like a <a>UUID</a>) that could be bound to multiple underlying <a>verifiable
-data registries</a> over time, thus maintaining its persistence independent of
-any particular system. However, registering the same identifier on multiple
-<a>verifiable data registries</a> makes it extremely difficult to identify the
-authoritative version of a <a>DID document</a> if the contents diverge
-between the different <a>verifiable data registries</a>. It also greatly
-increases implementation complexity for developers.
+With regard to the first caveat, requesting parties are advised not to make
+assumptions about the permanence of the binding of a <a>DID</a> to a
+<a>DID subject</a> in the absence of <a>DID</a> assignment policies specified
```

Comment:
We never talk about what a DID assignment policy is. We should either define it or remove this portion of the text.

Reply:
@kdenhartog You have a good point. I am revising the PR to remove references to "assignment policies".